truffle-config.js

```diff
@@ -72,7 +73,6 @@ module.exports = {
     },
     grep,
   },
-  test_file_extension_regexp: /.*\.js$/,
```

Comment:
So this is the sneaky setting that made none of my tests run when I tried it!

Reply:
:D yeah it took me some time to find it.